test.sh

```diff
@@ -5,10 +5,25 @@ cd java ; fish-pepper ; cd ..
 
 docker build java/images/jboss/ -t fabric8/s2i-java
 
-s2i build --copy java/images/example fabric8/s2i-java fabric8/s2i-java-example
+# Initially intentionally building without incremental
+s2i build --copy java/example fabric8/s2i-java fabric8/s2i-java-example
 
-docker run -p 8080:8080 fabric8/s2i-java-example
+# Now rebuild incrementally, it should not re-download .m2
+s2i build --copy java/example fabric8/s2i-java fabric8/s2i-java-example --incremental
 
-# TODO how to do simplest possible HTTP GET http://localhost:8080 and fail/pass?
-# TODO how to best stop docker again
-# kill $!
+CONTAINER_ID=$(docker run -d -p 8080:8080 fabric8/s2i-java-example)
+
+# sleep 1s is required because after docker run returns, the container is up but our server may not quite be yet
+sleep 1
```

Comment:
Is 1s good enough ?

Reply:
It "works for me", but let me make it 5s instead of 1s just to be on the safe side...